_includes/sections/work-we-do/sharing-our-expertise.md

```diff
@@ -1,12 +1,24 @@
 ### Sharing our expertise
 
+#### Hosting events
+
+We host a variety of face to face and online events to share our experience and
+expertise.
+
+We work hard to design events that are accessible, inclusive and safe. We ask
+attendees about their individual access and support needs. We publish a code of
+conduct for every event we host, and require one from every event we are
+involved with.
```

Comment:
> require one from every event we are involved with

Is this true, yet?

Reply:
I take this to mean that it _will_ be true for every event dxw get involved with from now on.

Reply:
Given Wendy's reply (https://github.com/dxw/playbook/pull/435#issuecomment-679083410), maybe we want to replace "require" with a less strong word.

Could we clarify what "involved with" means? If it's an event we sponsor, we could require a code of contact in a way we might not be able to if it's an event where we only participate by having someone speak or present.